### PR TITLE
Expo prep

### PR DESCRIPTION
**Pull request by barrettk**
- Add a basic 'getting started' vignette to showcase the core features
 - Fix broken links in README, and provide more background information on the package
 - Housekeeping ahead of adding `mrgda` to the Expo
    - Add `pkdown` site to DESCRIPTION file
    - add badges to README
    - Run `covr` during package release (hook up to drone)

closes https://github.com/metrumresearchgroup/mrgda/issues/169

### CODE REVIEW

**Comment by barrettk** [comment]
@michaelmcd18 @andersone1 I think it could be worth redeploying the `pkgdown` site _before_ this is merged (after it's approved) (`pkgdown::deploy_to_branch()`) so we have an updated front page with links to the vignettes (along with the new one). I also want to make sure the vignette links in the README work as intended. We followed a similar approach for `pmforest` so it should be fine.

@seth127 I deleted the `codecov.yml` (`bbr` had it) given your notes about coverage being low priority (and I dont know how that file worked). However, I did leave the drone additions and badges in the README. Not sure if anything else would need to be done outside of `mrgda` for this to work properly (like the registration for CI/CD), so just wanted to run that past you before merging.

**Comment by barrettk** [comment]
Preview of new README:
image

File: README.Rmd [hunk 1]
```diff
@@ -1,66 +1,68 @@
...
-mrgda is a data assembly helper, providing a set of functions that help 
+`mrgda` is a data assembly helper, providing a set of functions that help 
 you assemble, explore and validate your data set.
 
-An example data assembly script highlighting many `mrgda` functions can be found
-[here](https://ghe.metrumrg.com/templates/msproject/blob/master/script/data-assembly.Rmd).
 
+### Featured Vignettes
+
+* [Getting Started](https://metrumresearchgroup.github.io/mrgda/articles/getting-started.html) -- Basic examples and introduction to a typical data assembly workflow.
```
Comment (michaelmcd18) [inline README.Rmd:31]: Replace `Basic examples and introduction to a typical data assembly workflow` with something along the lines of `introduction to commonly used data assembly mrgda functions`

File: README.md [hunk 2]
```diff
@@ -1,25 +1,49 @@
...
+help you assemble, explore and validate your data set.
+
+### Featured Vignettes
 
-An example data assembly script highlighting many `mrgda` functions can
-be found
-[here](https://ghe.metrumrg.com/templates/msproject/blob/master/script/data-assembly.Rmd).
+-   [Getting
+    Started](https://metrumresearchgroup.github.io/mrgda/articles/getting-started.html)
+    – Basic examples and introduction to a typical data assembly
```
Comment (michaelmcd18) [inline README.md:22]: Remove `introduction to a typical data assembly`

**Comment by michaelmcd18** [comment]
@barrettk The vignette looks great! Just a couple small comments above with how we introduce it in the README

**Comment by barrettk** [comment]
@michaelmcd18 Just made the updates. I'll update the `pkdown` pages before merging if everything else looks good

**Comment by michaelmcd18** [comment]
@barrettk Everything looks great!

**Comment by barrettk** [comment]
@michaelmcd18 Can you approve the PR?

**Review by michaelmcd18 — APPROVED** [review]

**Pull request merged**